File: PasswordValidation/PasswordValidation.Tests/CompositePasswordValidatorShould.cs
using FluentAssertions;
using Xunit;

namespace PasswordValidation.Tests;

public class CompositePasswordValidatorShould
{
    [Fact]
    public void ReturnValidWhenUsingTheTestPasswordValidator()
    {
        var validators = new List<IPasswordValidator>
        {
            new TestPasswordValidator()
        };
        var validator = new CompositePasswordValidator(validators);
        validator.Validate("Ab1_xxxxx").Should().BeTrue();
    }
}

File: PasswordValidation/py/tests/composite_password_validator_should.py
from password_validation.composite_password_validator import CompositePasswordValidator
from tests.always_true_password_validator import AlwaysTruePasswordValidator


def test_return_true_when_using_the_always_true_validator():
    validators = [AlwaysTruePasswordValidator()]
    assert CompositePasswordValidator(validators).validate("Password_1") is True
